### PR TITLE
Feature/89 mask username on consent withdrawl

### PR DESCRIPTION
**Pull request by danieldelhoyo**

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

**Comment by vroldanbet** [comment]
:+1:

File: src/main/java/org/zalando/planb/provider/ConsentController.java [hunk 1]
```diff
@@ -19,12 +19,12 @@
 
 import org.zalando.planb.provider.api.Consent;
 import org.zalando.planb.provider.api.ConsentsApi;
+import org.zalando.planb.provider.realms.UserRealm;
 
 @RestController
+@Log4j2
```
Comment (vroldanbet) [inline src/main/java/org/zalando/planb/provider/ConsentController.java]: @danieldelhoyo please use @Slf4j instead

Comment (vroldanbet) [inline src/main/java/org/zalando/planb/provider/ConsentController.java]: too late :P

**Pull request merged**